boot/zephyr/CMakeLists.txt

```diff
@@ -141,16 +142,9 @@ zephyr_library_sources(
 endif()
 
 if(CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256 OR CONFIG_BOOT_ENCRYPT_EC256)
-  zephyr_library_include_directories(
-    ${MBEDTLS_ASN1_DIR}/include
-    )
-  zephyr_library_sources(
-    # Additionally pull in just the ASN.1 parser from mbedTLS.
-    ${MBEDTLS_ASN1_DIR}/src/asn1parse.c
-    ${MBEDTLS_ASN1_DIR}/src/platform_util.c
-    )
+	zephyr_include_directories(include)
   if(NOT DEFINED CONFIG_BOOT_USE_MBEDTLS)
-    message(FATAL_ERROR "Not using mbedtls")
+    #message(FATAL_ERROR "Not using mbedtls")
```

Comment:
why we commented this line?? are we not defining CONFIG_BOOT_USE_MBEDTLS?

Reply:
this is commented as when we use Tincrypt library (key based) we wont enable mbedtls and in that case this will give false error 
We use mbedtls in X509 based approach and Tincrypt in key based approach

Reply:
need cleaner way, you can modify it to proper one like if(CONFIG_BOOT_USE_TINYCRYPT)

Reply:
this was added by dawid (orignally the owner ) i will update the same when integrating with 3.0.0